src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/AggregateProcessPPIO.java

```diff
@@ -33,6 +37,10 @@ protected MapperWrapper wrapMapper(MapperWrapper next) {
             };
         };
         xstream.allowTypes(new Class[] { AggregateProcess.Results.class });
+        xstream.omitField(AggregateProcess.Results.class, "aggregateAttribute");
```

Comment:
Btw, I have copied the omits and committed to get the build working again, but this is just a stop-gap measure,  imho if there is no grouping you should leave the extra fields null, or, as an alternative, write a custom xstream converter that will handle the grouping extra attributes only when grouping is actually used.